iaso/models/org_unit.py

```diff
@@ -548,6 +549,37 @@ def source_path(self):
     def get_reference_instances_details_for_api(self) -> list:
         return [instance.as_full_model() for instance in self.reference_instances.all()]
 
+    def get_extra_fields(self):
+        from iaso.models import Account
+        from iaso.models.data_store import JsonDataStore
+
+        try:
+            datastore = self.jsondatastore_set.get(
+                account=Account.objects.filter(default_version=self.version).first(),
+                slug="extra_fields",
+            )
+            return datastore.content
+        except JsonDataStore.DoesNotExist:
+            return {}
+
+    def set_extra_fields(self, content):
+        from iaso.models import Account
+        from iaso.models.data_store import JsonDataStore
+
+        try:
+            datastore = self.jsondatastore_set.get(
+                account=Account.objects.filter(default_version=self.version).first(),
+                slug="extra_fields",
+            )
+            datastore.content = {**datastore.content, **content}
```

Comment:
TIL: the second dictionary's values overwriting those from the first!
It might be worth a comment and small test?

Reply:
Yep I love that splat thingy. And you're right! I'll add a test :)